examples/update-nfts-metadata-of-non-fungible-token.js

```diff
@@ -103,10 +103,10 @@ async function main() {
         nftInfo = tokenNftsInfo[0];
         console.log(`Set token NFT metadata:`, nftInfo.metadata);
 
-        // Create an account to transfer the NFT to
+        // Create an account to transfer the NFT to with 10 automatic token association slots
```

Comment:
Can we improve this comment, may be something like this:
```suggestion
        // Create an account with 10 automatic token association slots to transfer the NFT to
```

Reply:
sounds good to me